### PR TITLE
Refactored snap pkg

### PR DESCRIPTION
**Pull request by shikharvashistha**
Greetings of the day @guyluz11,

I've refactored snap package please review the PR and let me know in case of any changes.

Thanks & Regards
@shikharvashistha 

https://github.com/CyBear-Jinni/cbj_app/issues/273

### CODE REVIEW

**Comment by guyluz11** [comment]
Tested it and it does not work.

The other snap use 256x256 image so use [this one ](https://user-images.githubusercontent.com/9304740/132479923-a94aa095-3cf5-4e08-a2ea-d3f9593144e2.png), and please download it using the right-clicked "Save Image As..."

image

I think that the last image that you have add contains a white background.

I have tested it with 256x256 image and it still not working (very weird).

I suggest opening question about it in [https://forum.snapcraft.io/](https://forum.snapcraft.io/).

**Comment by guyluz11** [comment]
Found it, after going through [flutter doc](https://flutter.dev/docs/deployment/linux) I have found out that `Exec=` need to use the name in the `apps:` and not the name of the executable.

So `Exec=CyBear-Jinni_App` should be `Exec=cybear-jinni`.

image

File: snap/gui/cybear-jinni.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=cybear-jinni
+Comment=App to control your smart device through CyBear Jinni Hub
+Exec=CyBear-Jinni_App
```
Comment (guyluz11) [inline snap/gui/cybear-jinni.desktop]: Need to change to `Exec=cybear-jinni`

File: snap/gui/cybear-jinni.desktop [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=cybear-jinni
```
Comment (guyluz11) [inline snap/gui/cybear-jinni.desktop]: Can be changed to `CyBear Jinni`

File: snap/gui/cybear-jinni.desktop (hunk 1 above)
Comment (shikharvashistha) [reply snap/gui/cybear-jinni.desktop]: done

**Review by guyluz11 — COMMENTED** [review]
This and change the image to 256x256 as I said before and we are done :D

@shikharvashistha

**Comment by shikharvashistha** [comment]
> This and change the image to 256x256 as I said before and we are done :D
> 
> @shikharvashistha

Greetings of the day @guyluz11,

I've been using the 256x256 image from beginning. This  is for information please.

Thanks & Regards
@shikharvashistha

File: snap/gui/cybear-jinni.desktop (hunk 2 above)
Comment (shikharvashistha) [reply snap/gui/cybear-jinni.desktop]: Done ! 👍

**Comment by guyluz11** [comment]
> > This and change the image to 256x256 as I said before and we are done :D
> > @shikharvashistha
> 
> Greetings of the day @guyluz11,
> 
> I've been using the 256x256 image from beginning. This is for information please.
> 
> Thanks & Regards
> @shikharvashistha

hmm I see, so can you please upload the image again?, my test shows that it is 460x460 image with white background

image

**Review by guyluz11 — COMMENTED** [review]
One file good to go

**Comment by shikharvashistha** [comment]
> One file good to go

done ! 👍

**Comment by guyluz11** [comment]
Looks good.
Will merge it now

**Pull request merged**